Images_And_Graphics_style.css_v1

```diff
@@ -17,29 +17,36 @@ h1 {
 
 h2 {
   color: #000000;
-  font-size: 30px;
+  font-size: 20px;
   text-align: center;
 }
 
 h3 {
   color: #FFF;
-  font-size: 45px;
+  font-size: 26px;
   text-align: center;
-  margin-top: 30px;
+  margin-top: 20px;
   margin-left: 150px;
 }
 
 h4 {
   color: #FFF;
-  font-size: 45px;
+  font-size: 26px;
+  text-align: center;
+  margin-top: 20px;
+  margin-left: 1375px;
+}
+
+h5 {
+  color: #000000;
+  font-size: 30px;
   text-align: center;
-  margin-top: 30px;
-  margin-left: 1100px;
+  margin-left: 500px;

```

Comment:
Got rid of margin-top and changed margin-left from 1100px to 500px as the text kept going off box6 in the footer.